config/webpack/environment.js

```diff
@@ -62,4 +51,9 @@ Object.assign(fileLoader.use[0].options, {
   postTransformPublicPath: (p) => `window.rails_asset_host + ${p}`
 });
 
+// Whatever this is, it seems we don't need it.
+environment.loaders.delete('nodeModules')
```

Comment:
Probably this gives a little explanation:
https://github.com/rails/webpacker/blob/54c3ca9245e9ee330f8ca63b447c202290f7b624/docs/v4-upgrade.md#excluding-node_modules-from-being-transpiled-by-babel-loader

Reply:
You know what, this in fact should not be part of the PR. I removed it at first to "see" what happens but this loader comes from webpacker so why should we delete it. I will restore this line.

Reply:
If it isn't needed, then it can speed up compilation to be removed. So maybe keep that line. But as you decide.